webmessaging/multi-globals/messageport-current.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Making the current page become non-active must prevent message transmission</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<!-- This is the entry global -->
+
+<iframe src="support/incumbent.html" id="incumbent"></iframe>
+<iframe src="support/current.html" id="current"></iframe>
+
+<script>
+"use strict";
+setup({ explicit_done: true });
+
+const incumbentIframe = document.querySelector("#incumbent");
+const currentIframe = document.querySelector("#current");
+
+window.onload = () => {
+  promise_test(async t => {
+    const messageChannel = frames[0].createMessageChannel();
```

Comment:
Maybe use incumbentIframe.contentWindow for clarity? Otherwise there's no real reason to const it above.

Reply:
And maybe add a comment that this invokes the constructor in current.